### DISCUSSION TITLE
SendResponseAsync length

### DISCUSSION THREAD

**Discussion by matevzkovacic** [scottoffen/grapevine#151] [Q&A]
Hello,

I have simple sample to return response though await context.Response.SendResponseAsync(json);

If json is larger then 2048 characters Grapevine server crash and response is Pending on client.

Where can I extend size of response, I would like to send larger files?

![image](https://github.com/user-attachments/assets/c356a8d6-e1d1-41ad-b637-b8756b1eec72)

**Comment by scottoffen** [comment]
I am unable to duplicate this issue.
